src/components/features/ctf-components/ctf-page/ctf-page.graphql

```diff
@@ -1,62 +1,21 @@
 fragment PageTopSectionFields on PageTopSectionItem {
   __typename
-  ... on ComponentCta {
```

Comment:
what's the reason for the fragment removal?

Reply:
the internalName is no longer needed (was only for x-ray) so we could remove a lot of fragments 🎉